uktrade/data-workspace-frontend
# Fix ckeditor in admin for reference datasets

### Description of change
This change fixes the ckeditor in admin for the reference datasets. We found 2 issues:

1. jQuery was undefined as the init file was being loaded too late.
2. The script in the extended admin template wasn't firing due to "[nonce](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/nonce)"

jQuery is now defined and we have moved the inline script to an external js file.

### How to test
1. Open `/admin`
2. Choose any ref dataset
4. Check that the ckeditor is working


### Checklist
